eng/Dependencies.props

```diff
@@ -141,15 +141,8 @@ and are generated based on the last package release.
 
   <ItemGroup Label=".NET team dependencies (Non-source-build)" Condition="'$(DotNetBuildSourceOnly)' != 'true'">
     <LatestPackageReference Include="Microsoft.AspNet.WebApi.Client" />
-    <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.2.1" />
```

Comment:
Should 2.1 be removed?

Reply:
I think so, it contains only `netcoreapp2.1` assets, which isn't a supported TFM anymore. Plus it's not on the list of supported packages: https://dotnet.microsoft.com/en-us/platform/support/policy/aspnet/2.1-packages